examples/aml/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/getyoti/yoti-go-sdk"
+
+	_ "github.com/joho/godotenv/autoload"
+)
+
+var sdkID = os.Getenv("YOTI_CLIENT_SDK_ID")
+var key, err = ioutil.ReadFile(os.Getenv("YOTI_KEY_FILE_PATH"))
```

Comment:
I would do this as 
```
var (
  sdkID string
  key ioutil.File
  client *yoti.Client
)
```
and then initialise in main()
That way you can catch and handle the error